aiida/engine/transports.py

```diff
@@ -101,9 +100,6 @@ def do_open():
         try:
             transport_request.count += 1
             yield transport_request.future
```

Comment:
Just a quick check: this really needs to be yielded and not awaited?

Reply:
I think yes, this is the `yield` of contextmanager.